dali/pipeline/operator/op_schema.h

```diff
@@ -28,12 +28,25 @@
 #include "dali/core/format.h"
 #include "dali/core/traits.h"
 #include "dali/core/error_handling.h"
+#include "dali/pipeline/data/types.h"
 #include "dali/pipeline/operator/argument.h"
+#include "dali/pipeline/operator/copy_vector_helper.h"
 
 namespace dali {
 
 class OpSpec;
 
+struct RequiredArgumentDoc {
+  std::string doc;
+  DALIDataType dtype;
+};
+
+struct DefaultedArgumentDoc {
+  std::string doc;
+  DALIDataType dtype;
+  Value *default_value;
```

Comment:
This is not just "doc" - it's a vital part of the argument. Can you rename this structure?

Reply:
I went with `DefaultedArgumentDef`.